README.md

```diff
@@ -33,6 +33,81 @@ Python | macOS | Xcode | iPadOS, iOS |
 :------:|:------:|:------:|:------:|
 3.8 | 13.1 | 14.2 | 16.2 |
 
+## <a name="using-converted-weights"></a> Using Converted Weights from Hugging Face Hub
+
+<details>
+  <summary> Click to expand </summary>
+
+🤗 Hugging Face ran the [conversion procedure](#converting-models-to-coreml) on the following models and made the Core ML weights publicly available in the Hub:
+
+* [`CompVis/stable-diffusion-v1-4`](https://huggingface.co/apple/coreml-stable-diffusion-v1-4)
+* [`runwayml/stable-diffusion-v1-5`](https://huggingface.co/apple/coreml-stable-diffusion-v1-5)
+* [`stabilityai/stable-diffusion-2-base`](https://huggingface.co/apple/coreml-stable-diffusion-2-base)
+
+If you want to use any of those models you may download the weights and proceed to [generate images with Python](#image-generation-with-python) or [Swift](#image-generation-with-swift).
+
+There are several variants in each model repository. You may clone the whole repos using `git` and `git lfs`, or select just the variants you need. For example, to do generation in Python using the `ORIGINAL` attention implementation (read [this section](#converting-models-to-core-ml) for details), you could use the following helper code:
+
```

Comment:
NIT: It might be more useful to put the git instructions ahead of the alternative Python instructions because I think developers who would use ready-made models would appreciate being fully separated from the Python workflow based on some early feedback we got.